### PR TITLE
translate big_models.md and performance.md to chinese

### PR DESCRIPTION
**Pull request by jiaqiw09**
# What does this PR do?

Part of  #26803 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests? _not necessary_

## Who can review?
@stevhliu

### CODE REVIEW

**Comment by jiaqiw09** [comment]
@stevhliu

Hi, I will translate `Performance and scalability` part next.

For the possible merge conflict, I will fix it later after another pr merged. 

Best

File: docs/source/zh/big_models.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# 实例化大型模型
+
+当你想使用一个非常大的预训练模型时，一个挑战是尽量减少对RAM的使用。通常从PyTorch开始的工作流程如下：
```
Comment (statelesshz) [inline docs/source/zh/big_models.md:19]:
```suggestion
当你想使用一个非常大的预训练模型时，一个挑战是尽量减少对内存的使用。通常从PyTorch开始的工作流程如下：
```

File: docs/source/zh/performance.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+* [使用Trainer API进行超参数搜索](hpo_train)
+
+
+## 推理
+
+在生产环境中对大型模型进行高效推理可能与训练它们一样具有挑战性。在接下来的部分中，我们将详细介绍如何在CPU和单/多GPU设置上进行推理的步骤。
+
+* [在单个CPU上进行推理](perf_infer_cpu)
+* [在单个GPU上进行推理](perf_infer_gpu_one)
+* [多GPU推理](perf_infer_gpu_many)
```
Comment (stevhliu) [inline docs/source/zh/performance.md:49]: The `perf_infer_gpu_many` doc has been merged into the `perf_infer_gpu_one` doc now :)

```suggestion
* [多GPU推理](perf_infer_gpu_one)
```

**Review by stevhliu — APPROVED** [review]
Nice job! 👏

**Comment by jiaqiw09** [comment]
@stevhliu

Hi, I have fixed redirect problem.

Best

**Pull request merged**